__tests__/app.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const endpoints = require("../endpoints.json");
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe("Topics API", () => {
+	describe("Get Request", () => {
+		test("Get: 200, should return all topics", () => {
+			return request(app)
+				.get("/api/topics")
+				.expect(200)
+				.then(({ body }) => {
+					const { topics } = body;
+					expect(topics).toBeInstanceOf(Array);
+					expect(topics.length).not.toBeLessThan(1);
+					topics.forEach((topic) => {
+						expect(Object.keys(topic)).toEqual(["slug", "description"]);
+					});
+				});
+		});
+	});
```

Comment:
- Good test description which includes the expected status code.
- Checking it's an array, the length is not 0 and it contains the expected keys is good, well done.